pkgs/by-name/hy/hyfetch/package.nix

```
@@ -30,6 +31,13 @@ python3Packages.buildPythonApplication rec {
     "hyfetch"
   ];
 
+  postFixup = ''
+    mv $out/bin/neowofetch $out/bin/neowofetch-unwrapped
+
+    makeWrapper $out/bin/neowofetch-unwrapped $out/bin/neowofetch \
+      --prefix PATH : ${lib.makeBinPath [ pciutils ]}
```

```suggestion
    wrapProgram $out/bin/neowofetch \
      --prefix PATH : ${lib.makeBinPath [ pciutils ]}
```

Not sure why is that